libcxx/test/std/numerics/c.math/cmath.pass.cpp

```diff
@@ -1113,6 +1116,44 @@ void test_fmin()
     assert(std::fmin(1,0) == 0);
 }
 
+struct TestHypot3 {
+    template <class Real>
+    static void operator()() {
```

Comment:
```suggestion
    void operator()() const {
```

More friendly to older Standards!

Reply:
Fixed in a80b11e49e67a816c6c7cb116ea155ad9957764b.